qa/tasks/cephfs/test_sessionmap.py

```diff
@@ -230,3 +230,44 @@ def test_session_evict_blocklisted(self):
 
         self.mount_a.kill_cleanup()
         self.mount_a.mount_wait()
+
+    def test_session_evict_non_blocklisted(self):
+        """
+        Check that mds evicts without blocklisting client
+        """
+
+        self.fs.set_ceph_conf('mds', 'mds session blocklist on evict', False)
```

Comment:
shouldn't this be `mds_session_blocklist_on_evict` instead of `mds session blocklist on evict`?

Reply:
This is fine because the parser will help connect them. I am just following the existing code.